test/desktop_sikuli/tests/test_profile.py

```diff
@@ -6,13 +6,15 @@
 
 class TestProfile(BaseTestCase):
 
+    @pytest.mark.testrail_id(5590)
```

Comment:
I would expect that check for QR code will be added and test will be failed until fix for https://github.com/status-im/status-react/issues/6124

Reply:
This verification will be added after including of pytesseract library